src/vivarium_census_prl_synth_pop/tools/jobmon.py

```diff
@@ -53,27 +54,31 @@ def run_make_results_workflow(
             "stderr": str(final_output_dir),
         },
         default_cluster_name="slurm",
+        # Build the cli command to be run. The spaces after each part or required
         command_template=(
             f"{shutil.which('jobmon_make_results_runner')} "
             "{raw_output_dir} "
             "{final_output_dir} "
             "{verbose} "
             "{mark_best} "
             "{test_run} "
+            "{seed} "
             "--artifact-path {artifact_path} "
         ),
-        node_args=[],
+        # node_args=["seed"],  # TODO: parameterize by seed
```

Comment:
What is a node arg?

Reply:
It's the thing that we will be parameterizing by (like,  nodes in the dag)

https://scicomp-docs.ihme.washington.edu/jobmon/current/core_concepts.html#node-args